components/3002-call-undefined-root-component/error.txt

```diff
@@ -0,0 +1 @@
+Template "app-main" is used but it's not defined
```

Comment:
This should not be a compile-time error as the template may be defined in another file which might be compiled later or loaded dynamically.

It should not be a runtime error either, as it's possible that the server-side might want to render an empty HTML5 custom element which is then expanded client-side only.